harvard-huit/Harvard-Geospatial-Library
# Updated leaflet map colors

**Updated leaflet map colors**
* * *

**JIRA Ticket**: [HGL-345: Apply library design styles](https://jira.huit.harvard.edu/browse/HGL-345)

# What does this Pull Request do?
Incorporates new accessible color palette for leaflet map.

# How should this be tested?
Test cases:
* https://localhost:3001/catalog/stanford-fb897vt9938 (larger tiles)
* https://localhost:3001/catalog/harvard-g5831-r3-s20-f7-index-gr (smaller tiles)

The interactive paths on index maps will now have the following:
* Black stroke color around every tile path
* Fill-opacity of each tile is now darker (from 0.2 to 0.65), but you can still see outlines/words on the map underneath the colors
* Default color (used to be green): #7FCDBB
* Unavailable color (used to be red): #EDF8B1
* Selected color (used to be blue): #2C7FB8 (still blue, just a different shade)

Notes: 
* The "click to wake" button adds an opacity layer to the entire map, so colors appear darker after waking the map
* The accessibility tools for firefox has a [color blind simulator](https://developer.mozilla.org/en-US/docs/Tools/Accessibility_inspector/Simulation) to confirm the color combinations are appropriate

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? no
- integration tests? no

# Interested parties
@phil-plencner-hl 
@enriquediaz
